tool/src/js/runtime.mjs

```diff
@@ -5,7 +5,7 @@ export function readString8(wasm, ptr, len) {
 
 export function readString16(wasm, ptr, len) {
   const buf = new Uint16Array(wasm.memory.buffer, ptr, len);
-  return String.fromCharCode(buf)
+  return String.fromCharCode.apply(null, buf)
```

Comment:
heh, I noticed this earlier and figured this was just some functionality of `.fromCharCode` I didn't know, JS has a lot of these helpers. Should have left a comment.

Anyway, this works